ynput/ayon-core
# Tray: Send messages to tray

## Changelog Description
Added option to send request to show message in tray and use it when tray is already running.

## Additional info
This may not work if OS does not support tray messages. Not sure what are the cases, I think it can happen only if it does not support even sys tray, which is bigger issue.

## Testing notes:
1. Start tray.
1. Try to start tray again.
2. A tray message should show up that tray is already running.
